packages/morph/src/page/Editor/MarkdownRenderer.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  Decoration,
+  DecorationSet,
+  ViewPlugin,
+  ViewUpdate,
+  WidgetType,
+  EditorView
+} from "@codemirror/view";
+import { RangeSetBuilder } from "@codemirror/state";
+import { unified } from "unified";
+import remarkParse from "remark-parse";
+import remarkHtml from "remark-html";
+
+const mdProcessor = unified().use(remarkParse).use(remarkHtml);
```

Comment:
you probably need to lazily load the processor and use this inside the build phase.

In this case, the processor will be run every time the editor is open.

you would need to do remarkParse -> [md plugins] -> remarkRehype -> [html plugins] -> htmlToJsx (you can take a look at quartz docs structure, sorta similar inside quartz/build.ts)